src/linear_algebra/std_basis.lean

```diff
@@ -13,11 +14,23 @@ elsewhere.
 
 To give a concrete example, `std_basis R (λ (i : fin 3), R) i 1` gives the `i`th unit basis vector
 in `R³`, and `pi.is_basis_fun` proves this is a basis over `fin 3 → R`.
+
+## Main definitions
+
+ - `std_basis R ϕ i b`: the `i`'th standard `R`-basis vector on `Π i, ϕ i`
```

Comment:
Perhaps add "scaled by b"?

Reply:
It's not exactly scaled since `φ` is just a semimodule. I'll use that for now unless someone has a nice alternative.

Reply:
Oh, I see your point - the semimodule may not have a `1`.